src/istgt_lu.c

```diff
@@ -1710,6 +1710,22 @@ istgt_lu_add_unit(ISTGT_Ptr istgt, CF_SECTION *sp)
 	ISTGT_TRACELOG(ISTGT_TRACE_DEBUG, "ReadOnly %s\n",
 	    lu->readonly ? "Yes" : "No");
 #ifdef REPLICATION
+	val = istgt_get_val(sp, "DesiredReplicationFactor");
```

Comment:
we need to read replicas list as well from conf file
and checks are required that it is equal to the replication factor

Reply:
Updated the code